src/components/cqlLibraryList/CqlLibraryList.tsx

```diff
@@ -360,6 +360,7 @@ export default function CqlLibraryList({ cqlLibraryList, onListUpdate }) {
                             handleOpen(cqlLibrary, e);
                           }}
                           data-testid={`view/edit-cqlLibrary-button-${cqlLibrary.id}`}
+                          aria-label={`Measure ${cqlLibrary.cqlLibraryName} version ${cqlLibrary.version} draft status ${cqlLibrary.draft} View / Edit`}
```

Comment:
I'm not sure why we add Measure for the library actions.

Reply:
Because we typo.
Thank you for catching that